### PR TITLE
PORTALS-1664: add custom icons to organization page.

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

**Comment by jay-hodgson** [comment]
Need to update SRC, publish a new version, and pull that into here before merging this PR.

File: package.json [hunk 1]
```diff
@@ -16,7 +16,6 @@
     "clone-deep": "^4.0.1",
     "node-sass": "4.12.0",
     "npm-run-all": "^4.1.5",
-    "plotly.js-basic-dist": "^1.47.3",
```
Comment (jay-hodgson) [inline package.json:19]: deleted duplicate (plotly.js-basic-dist import is also listed on line 32/33)

File: src/configurations/nf/organizations.ts [hunk 2]
```diff
@@ -81,9 +82,10 @@ export const organizationDetailsPage: SynapseConfig[] = [
         title: 'organizationName',
         type: SynapseConstants.ORGANIZATION,
         description: 'summary',
-        icon: 'logo',
+        icon: 'abbreviation',
```
Comment (jay-hodgson) [inline src/configurations/nf/organizations.ts:85]: column used for mapping

File: package.json (hunk 1 above)
Comment (emmanmills) [reply package.json:19]: sounds good

**Review by emmanmills — APPROVED** [review]

**Pull request merged**